core/src/main/kotlin/model/Documentable.kt

```diff
@@ -113,11 +113,6 @@ abstract class Documentable {
 
     override fun hashCode() = dri.hashCode()
 
-
-
-    val commentsData: List<DocumentationNode>
-        get() = platformInfo.map { it.documentationNode }
```

Comment:
We don't need this anymore as by referencing documentation nodes with this shortcut we lose information about specific platform data